frontend/ui/test/services/in_memory_trip_service_test.dart

```diff
@@ -16,19 +16,19 @@ void main() {
 
   group('Populated InMemoryTripService', () {
     var tripService = InMemoryTripService();
-    var trip1 = Trip(id: 'id1', name: 'name1');
-    var trip2 = Trip(id: 'id2', name: 'name2');
+    var trip1 = Trip(name: 'name1');
+    var trip2 = Trip(name: 'name2');
     // do async setup inside a setUp lambda because group lambdas cant be async
-    setUp(() async {
-      await tripService.createTrip(trip1);
-      await tripService.createTrip(trip2);
+    setUpAll(() async {
```

Comment:
I didn't realize `setUp` runs before each test, `setUpAll` runs for all tests in the group. So this is a fix of that too.